src/components/NewVaultOfferSummary.tsx

```diff
@@ -0,0 +1,69 @@
+import clsx from 'clsx';
+
+type TableRowProps = {
+  left: string;
+  right: string;
+};
+
+const TableRow = ({ left, right }: TableRowProps) => {
+  return (
+    <tr className="text-[13px] leading-[21px]">
+      <td className="text-[#A3A5B9]">{left}</td>
+      <td className="text-right text-[#666980] font-black">{right}</td>
+    </tr>
+  );
+};
+
+const NewVaultOfferSummary = () => {
+  const isDisabled = true;
```

Comment:
I suppose this is because the UI can't make new vault offers yet.

Reply:
Right, this will be determined by the state of the inputs eventually